src/components/Button.jsx

```diff
@@ -0,0 +1,13 @@
+const Button = ({ onClick, content, className }) => {
```

Comment:
This component is not needed because it has no unique properties. Its styling is handled in the "main.css" .